src/sidebar/components/SidebarView.js

```diff
@@ -91,9 +91,20 @@ function SidebarView({
 
   useEffect(() => {
     const framePort = new PortFinder();
+
+    // Enables communication between `host` and `sidebar` frames
+    framePort
+      .discover({
+        channel: 'hostToSidebar',
+        hostFrame: window.parent,
+        port: 'sidebar',
+      })
+      .then(port => frameSync.connect(port));
```

Comment:
Might make sense to move the `framePort` here into the `FrameSync` service to keep the pattern that all communication from parts of the sidebar to the host page and guests goes via that service. Alternatively we could have separate services for the sidebar to communicate with guests vs. communicating with non-guests.

Reply:
I totally agree.